src/backend/trace.rs

```diff
@@ -336,6 +374,18 @@ impl TraceData {
                 let dur = probe.duration();
                 let dur_ms = dur.as_secs_f64() * 1000_f64;
                 hop.total_time += dur;
+                //Before last is set use it to set jitter
+                let last_ms = hop.last_ms().map_or(0f64, |d|d);
+                let jitter_ms = (last_ms - dur_ms).abs();
+                let jitter_dur = Duration::from_millis(jitter_ms as u64);
+                hop.jitter = Some(jitter_dur);
```

Comment:
This is set to be `Some` unconditionally, I guess we should only set this if `hop.last` is `Some`?